### DISCUSSION TITLE
jemalloc issue on rpi5

### DISCUSSION THREAD

**Discussion by eratio08** [quickwit-oss/quickwit#4468] [General]
Just FYI quickwit and other service utilizing jemalloc are currently not working on the Raspberry Pi 5 if the 16K page size is used.
See https://github.com/raspberrypi/bookworm-feedback/issues/107 for a workaround (switch back to 4K page size)

The error message will look like
```shell
 <jemalloc>: Unsupported system page size
 <jemalloc>: Unsupported system page size
 memory allocation of 5 bytes failed
```

the workaround is to add the following to the `boot/config.txt`
```
kernel=kernel8.img
```

**Comment by nickkaltner** [comment]
btw issue here https://github.com/quickwit-oss/quickwit/issues/4785

**Comment by fulmicoton** [comment]
closing as duplicate
